### PR TITLE
Add blog post for Multicloud Kubernetes apps

### PR DESCRIPTION
**Pull request by metral**
### Proposed changes

Add blog post for multicloud Kubernetes apps

### CODE REVIEW

File: content/blog/multicloud-app/index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+tags: ["Kubernetes","EKS", "AKS", "GKE"]
+meta_desc: "Use Pulumi to run Kubernetes apps using a multicloud strategy. In this post, we'll walk through how your apps can leverage multiple Kubernetes providers for deployments that can span AWS, Azure, and GCP."
+date: "2019-08-13"
+
+meta_image: "multicloud.png"
+---
+
+Kubernetes clusters from the managed offerings of AWS EKS,
+Azure AKS, and GCP GKE all vary in configuration, management, and resource
+properties. This variance creates complexity in cluster provisioning and app
```
Comment (ericrudder) [inline content/blog/multicloud-app/index.md]:
```suggestion
properties. This variance creates unnecessary complexity in cluster provisioning and app
```

File: content/blog/multicloud-app/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+deployments, as well as for CI/CD and testing. 
+
+Additionally, if you wanted to deploy the *same* app across multiple clusters 
+for specific use cases or test scenarios across providers, subtleties
+such as LoadBalancer outputs, and cluster connection settings can be a nuisance
```
Comment (ericrudder) [inline content/blog/multicloud-app/index.md]:
```suggestion
such as LoadBalancer outputs and cluster connection settings can be a nuisance
```

File: content/blog/multicloud-app/index.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+file. For more information on Pulumi's Kubernetes support, check out the [Kubernetes reference page]({{< relref "/docs/reference/clouds/kubernetes" >}}).
+
+We will use the cloud SDKs to provision the managed Kubernetes clusters. Given
+that we're working with real code, we are afforded developer benefits such as: 
+code linting, type checking, IDE hints and completion, and object oriented
+constructs such as abstractions and inheritance.
+
+Leveraging these development features creates the opportunity to encapsualte
+the finer-grained details and settings, and expose the capability to create
+clusters as simple as the following:
```
Comment (ericrudder) [inline content/blog/multicloud-app/index.md]:
```suggestion
clusters as simple as the following code:
```

File: content/blog/multicloud-app/index.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+- Azure: [`pulumi/azure`](https://github.com/pulumi/pulumi-azure)
+
+[Crosswalk for AWS][crosswalk-aws] further allows us to leverage the Pulumi
+libraries of common infrastructure for AWS to simplify cloud resource
+instantiation and management while gaining best-practices as defaults.
+Check out the [`pulumi/awsx`](https://github.com/pulumi/pulumi-awsx) SDK to get
+started.
+
+For local clusters such as those that are self-managed, or provisioned by a
+tool like up by Docker Desktop, Pulumi can still deploy workloads to these these
```
Comment (lblackstone) [inline content/blog/multicloud-app/index.md]:
```suggestion
tool like Docker Desktop, Pulumi can still deploy workloads to these these
```

File: content/blog/multicloud-app/index.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+systems given that the [`pulumi/kubernetes`][pulumi-kubernetes] workload SDK only requires a valid `kubeconfig`
+file. For more information on Pulumi's Kubernetes support, check out the [Kubernetes reference page]({{< relref "/docs/reference/clouds/kubernetes" >}}).
+
+We will use the cloud SDKs to provision the managed Kubernetes clusters. Given
+that we're working with real code, we are afforded developer benefits such as: 
+code linting, type checking, IDE hints and completion, and object oriented
```
Comment (lblackstone) [inline content/blog/multicloud-app/index.md]:
```suggestion
code linting, type checking, IDE hints and completion,
```

File: content/blog/multicloud-app/index.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+constructs such as abstractions and inheritance.
```
Comment (lblackstone) [inline content/blog/multicloud-app/index.md]:
```suggestion
abstractions and inheritance.
```

File: content/blog/multicloud-app/index.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+As shown in the code samples, it becomes relatively easy to provision and
+manage Kubernetes clusters, as well as deploy workloads to the cluster
+regardless if they are managed by a cloud provider, or self-managed.
+
+The various SDKS allow you to leverage industry standard best-practices and 
+defaults, in addition to allowing you to further configure and customize how your clusters
+and apps are managed.
+
+Testing apps across various providers in this form allows you to obfuscate away
```
Comment (lblackstone) [inline content/blog/multicloud-app/index.md]:
```suggestion
Testing apps across various providers in this form allows you to abstract away
```

File: content/blog/multicloud-app/index.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+join our [Community Slack](https://slack.pulumi.com/) channel if you have
+any questions, need support, or just want to say hello.
+
+If you'd like to chat with our team, or get hands-on assistance with
+migrating your existing configuration code to Pulumi, please don't hesitate to [drop us a line]({{< relref "/contact" >}}).
+
+We also encourage you to watch Pulumi team member [Levi Blackstone][levi-blackstone]
+demo this post in an episode of the [Kubernetes Community Meeting](https://kubernetes.io/community).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EyW2m5Xa_BQ?start=67" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (cnunciato) [inline content/blog/multicloud-app/index.md]: We should use [the YouTube shortcode](https://github.com/pulumi/docs/blob/master/BLOGGING.md#video) for these:

```suggestion
{{< youtube "EyW2m5Xa_BQ?rel=0&start=67" >}}
```

(Ensures all the sizing happens correctly.)

**Pull request merged**